sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/basicOperators.scala

```diff
@@ -458,7 +458,9 @@ case class Limit(limitExpr: Expression, child: LogicalPlan) extends UnaryNode {
   }
 }
 
-case class Subquery(alias: String, child: LogicalPlan) extends UnaryNode {
+case class Subquery(alias: String, child: LogicalPlan)
+  extends UnaryNode {
```

Comment:
I had once modified `Subquery` and then reverted the essential change. Will revert this one.